images/image-user/Makefile

```diff
@@ -12,22 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all test-image-user-uid test-image-user-username test-image-user-uid-group test-image-user-username-group
+include ../../hack/make-rules/Makefile.manifest
+
+.PHONY: all test-image-user-uid test-image-user-username test-image-user-uid-group test-image-user-username-group push-manifest
+
+REGISTRY = gcr.io/cri-tools
+ALL_ARCH = amd64 ppc64le
```

Comment:
Add all arches that kubernetes has already supported? e.g. `amd64 arm arm64 ppc64le s390x`

Reply:
@feiskyer : Should we use arm32v7 and arm64v8 ? [Reference](https://github.com/docker-library/official-images#architectures-other-than-amd64)